src/main/java/com/softserve/academy/spaced/repetition/controller/dto/impl/DeckCreateValidationDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.softserve.academy.spaced.repetition.controller.dto.impl;
+
+import org.hibernate.validator.constraints.Length;
+
+public class DeckCreateValidationDTO {
+
+    @Length(min = 2, max = 50)
```

Comment:
Length is Hibernate specific. Please replace by javax.validation.constraints.Size.

Reply:
Please also take into account that according https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/Size.html null values are considered valid. Maybe it makes sense to add NotEnmpty or NotBlank from https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/package-summary.html

Reply:
I've added changes. Please review them.P.S.: there is a mistake in commit description-instead of DeckControllerTest.java should be DeckServiceTest.java.